### ISSUE TITLE
AttributeError: 'SyncRPCFilterRequestBuilder' object has no attribute 'limit'

### ISSUE THREAD

**Issue by davidwitten** [supabase-community/supabase-py#700]
**Describe the bug**

The Python client doesn't support limiting rows in an RPC call and fails with "AttributeError: 'SyncRPCFilterRequestBuilder' object has no attribute 'limit'"

**To Reproduce**
```
function_data = supabase.rpc("function", {"param": "a"}).limit(5).execute().data
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>>> AttributeError: 'SyncRPCFilterRequestBuilder' object has no attribute 'limit'
```

**Expected behavior**
Returns a list of the results of "function" limited to 5 rows.

This works in the JS client:
```
const { data } = await supabase.rpc(fxn, { param: 'a'}).limit(5);
```

**Comment by silentworks** [comment]
I can confirm this is a bug in the Python library.

**Comment by silentworks** [comment]
I have a version of this working locally but there seem to be a few more filters missing so I'm going to try and get them all added before creating a PR for this.

Missing: `select`, `single`, `maybeSingle`, `limit`, `explain`, `range`, `offset`, `order` and a few more.

**Comment by silentworks** [comment]
I've created a PR over in the `postgrest-py` project to get this issue fixed https://github.com/supabase-community/postgrest-py/pull/372

**Comment by silentworks** [comment]
This has now been merged and is available in the latest 2.3.8 version of the library on PyPi. https://pypi.org/project/supabase/

**Comment by silentworks** [comment]
Closing this out as there has been no update from OP and a fix has been out from 2 weeks ago.
